AGENTS.md

```
@@ -0,0 +1,62 @@
+# Reka UI Monorepo
+
+## Repo Map
+| Path | Purpose |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)